anomalib/utils/cli/cli.py

```diff
@@ -165,25 +162,24 @@ def __set_callbacks(self) -> None:
         #  TODO: This could be set in PostProcessingConfiguration callback
         #   - https://github.com/openvinotoolkit/anomalib/issues/384
         # Normalization.
-        normalization = config.post_processing.normalization_method
-        if normalization:
-            if normalization == "min_max":
-                callbacks.append(MinMaxNormalizationCallback())
-            elif normalization == "cdf":
-                callbacks.append(CdfNormalizationCallback())
-            else:
-                raise ValueError(
-                    f"Unknown normalization type {normalization}. \n" "Available types are either None, min_max or cdf"
-                )
+        # normalization = config.post_processing.normalization_method
+        # if normalization:
+        #     if normalization == "min_max":
+        #         callbacks.append(MinMaxNormalizationCallback())
+        #     elif normalization == "cdf":
+        #         callbacks.append(CdfNormalizationCallback())
+        #     else:
+        #         raise ValueError(
+        #             f"Unknown normalization type {normalization}.
+        #  \n" "Available types are either None, min_max or cdf"
+        #         )
```

Comment:
We are not going to remove these?

Reply:
I just commented these for the pre-commit to work. I will remove these in the CLI PR